main.go

```diff
@@ -1,35 +0,0 @@
-package main
-
-import (
-	"log"
-	"os"
-
-	"github.com/env0/terratag/cli"
-	"github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/logutils"
-)
-
-func main() {
-	args, isMissingArg := cli.InitArgs()
-	if isMissingArg {
-		return
-	}
-	initLogFiltering(args.Verbose)
-
-	cli.Terratag(args)
-}
-
-func initLogFiltering(verbose bool) {
-	level := "INFO"
```

Comment:
why this file deleted?

Reply:
Thanks for catching this!!
it was omitted due to .gitignore.
I fixed .gitingore.
It's now under ./cmd

bf20b8175cd2b3c5522387396afd97baa67587f7